functions.php

```diff
@@ -105,6 +105,17 @@ function cc_mime_types($mimes) {
  */
 add_action('tgmpa_register', 'adventist_register_required_plugins');
 function adventist_register_required_plugins() {
+    if (get_bloginfo('version') >= '5.0.1') {
+      $plugin_name = 'Classic Editor';
+      $plugin_slug = 'classic-editor';
+      $plugin_required = false;
+      $plugin_activation = false;
```

Comment:
Let's make this activated and required for now.

Reply:
Addresses both comments: cc753b0 & 2b43723